packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -261,6 +270,16 @@ const makeAuctionDriver = async (t, customTerms, params = defaultParams) => {
     getTimerService() {
       return timerService;
     },
+    async getScheduleTracker(t) {
```

Comment:
glad the subscribtionTracker was usable here.

note: this function synchronously returns a promise so no need for `async`